Mouseless input

nojronatron opened the pull request:
- [x] Set event listeners for '+', '-', and '/' keys to perform IN, OUT, DROP actions respectively.
- [x] Maintain usual click action on same buttons.
- [x] Update README.

See #66

nojronatron committed:
Update version.

nojronatron committed:
Add copilot instructions.

nojronatron committed:
Merge pull request #67 from nojronatron/dev-2025-updates

Commit:
Set up in, out, drop key events.

jhmartin commented:
Looking good! To allow for easier pecking on laptop keyboards that lack numpads, how about treating `=` the same as `+` so I don't have to hit shift?

Commit:
Update docs with expected changes.

Commit:
Added use of Global dir, fixes #63

jhmartin committed:
Allow = in addition to + for IN

Lacking a numpad, we allow an equal sign to be treated same as a plus to
skip having to hit SHIFT on every in.

jhmartin committed:
Update usage popup with shortcuts

jhmartin committed:
Update README with shortcuts.

jhmartin commented:
> Looking good! To allow for easier pecking on laptop keyboards that lack numpads, how about treating `=` the same as `+` so I don't have to hit shift?

Implemented in 4d362e26c273ea4a6e00dfcdfc5462043c129143

nojronatron commented:
Thanks Jason 🎉 I'll test drive it soon and merge in. 🚀

Commit:
Rename instructions so they work.

Commit:
Fix help btn and offscreen el tabbing.

nojronatron committed:
Merge pull request #70 from nojronatron/doc-updates

Update docs with expected changes.

Commit:
Merge branch 'staging' of https://github.com/nojronatron/Bigfoot-Bib-Report-WL-Form into patch-offscreen-el-visibility

nojronatron committed:
Merge pull request #72 from nojronatron/patch-offscreen-el-visibility

Patch offscreen el visibility

Commit:
Merge branch 'staging' of https://github.com/nojronatron/Bigfoot-Bib-Report-WL-Form into mouseless-input
